Career list scroll state

Joggz opened the pull request:
### What does this PR do?
-review for the scroll state effects for the career list page

Joggz committed:
adds career input and table section:
-mobile view and Desktop view

Joggz committed:
add the 404-section

Joggz committed:
adds the mobile view for the 404-section

Joggz committed:
review for scroll state effect on careerlist page

SEUNAGBEYE commented on pages/careerlist.html:
```diff
@@ -6,8 +6,146 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="../css/main.css">
     <title>Document</title>
```
Change this title to `Career`

Joggz replied:
### Done

Joggz committed:
adds a link to the last child of each table data:
- click on left arrow links to careerdetail page (for desktop);
- click on the job title links to careerdetail page (for mobile)

Joggz committed:
changes font-family

Joggz committed:
adds font-family

Joggz committed:
adds position and changed margin navbar

Joggz committed:
change title of the page to Career
